Title: Remove inline keyword for read_data and write_data
Date: 2015-06-28
Origin: https://chromium.googlesource.com/chromiumos%2Fthird_party%2Ftrousers/+/c9c7cd50640c2d8882a04f59f1bcb383a88b19e9
Bug-Debian: http://bugs.debian.org/778149
Index: trousers/src/include/tcsps.h
===================================================================
--- trousers.orig/src/include/tcsps.h
+++ trousers/src/include/tcsps.h
@@ -23,13 +23,6 @@ int		   get_file();
 int		   put_file(int);
 void		   close_file(int);
 void		   ps_destroy();
-#ifdef SOLARIS
-TSS_RESULT  read_data(int, void *, UINT32);
-TSS_RESULT  write_data(int, void *, UINT32);
-#else
-inline TSS_RESULT  read_data(int, void *, UINT32);
-inline TSS_RESULT  write_data(int, void *, UINT32);
-#endif
 int		   write_key_init(int, UINT32, UINT32, UINT32);
 TSS_RESULT	   cache_key(UINT32, UINT16, TSS_UUID *, TSS_UUID *, UINT16, UINT32, UINT32);
 TSS_RESULT	   UnloadBlob_KEY_PS(UINT16 *, BYTE *, TSS_KEY *);
Index: trousers/src/include/tspps.h
===================================================================
--- trousers.orig/src/include/tspps.h
+++ trousers/src/include/tspps.h
@@ -18,8 +18,8 @@
 
 TSS_RESULT	   get_file(int *);
 int		   put_file(int);
-inline TSS_RESULT  read_data(int, void *, UINT32);
-inline TSS_RESULT  write_data(int, void *, UINT32);
+TSS_RESULT	   read_data(int, void *, UINT32);
+TSS_RESULT	   write_data(int, void *, UINT32);
 UINT32		   psfile_get_num_keys(int);
 TSS_RESULT	   psfile_get_parent_uuid_by_uuid(int, TSS_UUID *, TSS_UUID *);
 TSS_RESULT	   psfile_remove_key_by_uuid(int, TSS_UUID *);
Index: trousers/src/tcs/ps/ps_utils.c
===================================================================
--- trousers.orig/src/tcs/ps/ps_utils.c
+++ trousers/src/tcs/ps/ps_utils.c
@@ -42,11 +42,7 @@
 struct key_disk_cache *key_disk_cache_head = NULL;
 
 
-#ifdef SOLARIS
 TSS_RESULT
-#else
-inline TSS_RESULT
-#endif
 read_data(int fd, void *data, UINT32 size)
 {
 	int rc;
@@ -64,11 +60,7 @@ read_data(int fd, void *data, UINT32 siz
 }
 
 
-#ifdef SOLARIS
 TSS_RESULT
-#else
-inline TSS_RESULT
-#endif
 write_data(int fd, void *data, UINT32 size)
 {
 	int rc;
